2022Swerve/src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -41,6 +45,12 @@ public class SwerveSubsystem extends SubsystemBase{
 
     private AHRS gyro = new AHRS(SPI.Port.kMXP);
 
+    private DataLog datalog = DataLogManager.getLog();
+    private DoubleLogEntry rotationOutputLog = new DoubleLogEntry(datalog, "/swerve/rotout"); //Logs rotation state output
+    private DoubleLogEntry translationXOutputLog = new DoubleLogEntry(datalog, "/swerve/txout"); //Logs x translation state output
+    private DoubleLogEntry translationYOutputLog = new DoubleLogEntry(datalog, "/swerve/tyout"); //Logs y translation state output
+    private StringLogEntry controlSchemeLog = new StringLogEntry(datalog, "/swerve/scheme"); //Logs if robot is in FOD/ROD
```

Comment:
Instead of logging this, we should consider whether we should log the change of the control scheme instead?

Reason I am thinking this, is that it should almost never change, and instead of repeatedly logging a string, we can just use the log message to determine when it changes.

Reply:
( The log message would just be in the command to change from one to the other.

Reply:
We are missing the inputs.  We are only logging the outputs?

Reply:
> log the change of the control scheme instead?
Fully agreed, however with our current control infrastructure this would be a cumbersome fix, that would be fixed anyway with the upcoming controls rework.

Reply:
> We are missing the inputs. We are only logging the outputs?

No, inputs are logged automatically via Robot.java